bluebrain/repo-patches/packages/steps/package.py

```diff
@@ -33,11 +33,13 @@ class Steps(CMakePackage):
                     "RelWithDebInfoAndAssert"))
     variant("caliper", default=False, description="Build in caliper support (Instrumentor Interface)")
     variant("likwid", default=False, description="Build in likwid support (Instrumentor Interface)")
+    variant("vesicle", default=True, when="@5:", description="Add vesicle model")
 
     depends_on("benchmark", type=("build", "test"))
     depends_on("boost", when="@:3")
     depends_on("boost", when="@4:", type="build")
     depends_on("blas")
+    depends_on("gsl", when="+vesicle", type="build")
```

Comment:
is `gsl` only `build` dependency? I assume it's linked somewhere while building. 🤔

Reply:
absolutely! Fixed in e595db4